apps/common/behaviours/pdf-upload.js

```diff
@@ -16,7 +16,9 @@ module.exports = superclass => class PDFUpload extends superclass {
           name: 'application_form.pdf',
           data: pdfBuffer,
           mimetype: 'application/pdf'
-        }).catch(err => next(new Error(err.body)));
+        }).catch(err => {
+          next(new Error(err));
+        });
```

Comment:
Probably worth logging an error.
Also, there's no guarantee that there will be a `body` property which could lead to `undefined` errors so you could do this
```suggestion
        }).catch(err => {
        req.log('error', `Error during PDF processing: ${err.message || err}`);
        // Pass a new Error to next, ensuring that err.body is checked and a fallback message is provided
        next(new Error(err.body || 'An unknown error occurred during PDF processing.'));
      });
```

Reply:
I have amended this now